torch_geometric/nn/conv/fused_gat_conv.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Tuple, Union
+
+from dgNN.operators import GATConvFuse
```

Comment:
Can we make this an optional dependency? Similar to [here](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/spline_conv.py#L15)?